Makefile.am

```diff
@@ -102,8 +102,6 @@ lib_libtarsnap_a_SOURCES=					\
 		lib/crypto/crypto_keys_subr.c			\
 		lib/crypto/crypto_passwd_to_dh.c		\
 		lib/crypto/crypto_rsa.c				\
-		lib/crypto/crypto_scrypt.c			\
```

Comment:
I'm less confident about this change than the other ones, because of `crypto_scrypt_smix*`.

- should they stay in `lib/` because they don't depend on `platform.h` ?
- should they move to `lib-platform` because `crypto_scrypt.*` is moving?
- or is this a sign that this whole idea needs a bit more thought and/or a different approach?

Reply:
I'm fine with leaving them in `/lib/`.